Core/Base/ConsumerModule.py

```diff
@@ -0,0 +1,81 @@
+"""
+  DIRAC class which is the abstract class for the consumers
+
+  All DIRAC consumers must inherit from the basic class ConsumerModule
+  and override its methods
+"""
+
+from DIRAC.WorkloadManagementSystem.Consumer.ConsumerTools import getConsumerOption
+from DIRAC.WorkloadManagementSystem.Consumer.ConsumerTools import getConsumerSection
```

Comment:
You are right. I moved it to Core/Utilities. However I think this should be refactored later as a bigger refactoring step of all those helper functions, which are spread out everywhere. But this a separate subject